5.5.0_2.2/Dockerfile

```diff
@@ -0,0 +1,82 @@
+FROM buildpack-deps:jessie
+
+MAINTAINER Jack <jack@expert360.com>
+
+# ====================
+# Install Nodejs 5.5.0 LTS
```

Comment:
judging from this link https://hub.docker.com/_/node/ it doesn't look like there is one for that version. Could we go higher even if they're out of sync?